### DISCUSSION TITLE
Does this PowerShell module work without depending on Azure DevOps and can we use any code repository like Github Enterprise for storing ADF code?

### DISCUSSION THREAD

**Discussion by himanshudbagupta** [Azure-Player/azure.datafactory.tools#366] [Q&A] [answered]
As this is a PowerShell module, does that mean that we don't need to use Azure DevOps at all and everything can be done using Powershell itself? In our ADF build and deployment process, we want to use Github Enterprise as code repository along with this Powershell module without depending on Azure DevOps. Is that possible?

**Comment by NowinskiK** [comment, marked as answer]
Yes, that's correct. Wherever you can use PowerShell - you can use this module to publish ADF.

**Reply by himanshudbagupta** [reply]
Thanks for the confirmation. It helps.
I will try this for our use case where we will use Github Enterprise and PowerShell without any dependency on Azure DevOps.
